languages/csharp/BitwardenClient.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Bitwarden.Sdk;
+
+public sealed class BitwardenClient : IDisposable
+{
+    private readonly ProjectsClient _projectsClient;
+    private readonly SecretsClient _secretsClient;
+    private readonly CommandRunner _commandRunner;
+    private readonly BitwardenSafeHandle _handle;
+
+    public BitwardenClient(BitwardenSettings? settings = null)
+    {
+        var clientSettings = new ClientSettings
+        {
+            ApiUrl = settings is { ApiUrl: not null } ? settings.ApiUrl : "https://api.bitwarden.com",
+            IdentityUrl = settings is { IdentityUrl: not null } ? settings.IdentityUrl : "https://identity.bitwarden.com",
+            DeviceType = DeviceType.Sdk,
+            UserAgent = "Bitwarden DOTNET-SDK"
+        };
+
+        _handle = BitwardenLibrary.Init(clientSettings.ToJson());
+        _commandRunner = new CommandRunner(_handle);
+        _projectsClient = new ProjectsClient(_commandRunner);
+        _secretsClient = new SecretsClient(_commandRunner);
+    }
+
+    public ResponseForApiKeyLoginResponse? AccessTokenLogin(string accessToken)
+    {
+        var command = new Command();
+        var accessTokenLoginRequest = new AccessTokenLoginRequest
+        {
+            AccessToken = accessToken
+        };
+        command.AccessTokenLogin = accessTokenLoginRequest;
+        return _commandRunner.RunCommand<ResponseForApiKeyLoginResponse>(command);
+    }
+
+    public ProjectsClient Projects()
+    {
+        return _projectsClient;
+    }
+
+    public SecretsClient Secrets()
+    {
+        return _secretsClient;
+    }
+
+    public void Dispose()
+    {
+        BitwardenLibrary.FreeMemory(_handle);
```

Comment:
Freeing memory is handled by the handle and calling it twice can be bad.

Reply:
fixed